integrationTests/testProcessorNode.go

```diff
@@ -210,9 +210,6 @@ const stateCheckpointModulus = 100
 // StakingV2Epoch defines the epoch for integration tests when stakingV2 is enabled
 const StakingV2Epoch = 1000
```

Comment:
why this constant is still here? you have aded StakingV2 epoch method

Reply:
this may be used to create the config for a real enable epochs handler component, long tests are handled on the last PR